common/services/prismic/seasons.ts

```diff
@@ -0,0 +1,55 @@
+import Prismic from 'prismic-javascript';
+import { getPage } from '@weco/common/services/prismic/pages';
+import { getArticles } from '@weco/common/services/prismic/articles';
+import { getBooks } from '@weco/common/services/prismic/books';
+import { getEvents } from '@weco/common/services/prismic/events';
+import { getExhibitions } from '@weco/common/services/prismic/exhibitions';
+import { SeasonWithContent } from '@weco/common/model/season';
+
+export async function getSeasonWithContent(
+  req: Request,
```

Comment:
I'm a bit confused about what this type should be - this is failing the build

Reply:
Seems to be alright for me. The Object type below is more the issue.
<img width="614" alt="Screenshot 2020-11-23 at 15 44 24" src="https://user-images.githubusercontent.com/31692/99982688-cc45a400-2da2-11eb-9eb0-af433ece4d8f.png">